docs/source/gbd2017_models/causes/ischemic_heart_disease/index.rst

```diff
@@ -147,22 +150,46 @@ State and Transition Data Tables
      - Measure
      - Value
      - Notes
-   * - C
-     - prevalence
-     - prevalence_c493
+   * - All
+     - cause-specific mortality rate
+     - :math:`\frac{\text{deaths_c493}}{\text{population}}`
+     -
+   * - P
+     - excess mortality rate
+     - emr_m15755
      -
-   * - C
+   * - A
      - excess mortality rate
-     - :math:`\frac{\text{deaths_c493}}{\text{population} \,\times\, \text{prevalence_c493}}`
+     - emr_m1814
      -
-   * - C
+   * - S
+     - excess mortality rate
+     - 0
+     -
+   * - P
      - disability weight
-     - :math:`\displaystyle{\sum_{s\in \text{sequelae_c493}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+     - :math:`\displaystyle{\sum_{s=380}^{385}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
      -
-   * - All
-     - cause-specific mortality rate
-     - :math:`\frac{\text{deaths_c493}}{\text{population}}`
+   * - A
+     - disability weight
+     - :math:`\scriptstyle{\text{disability_weight_378} \,\times\, \text{prevalence_378} + \text{disability_weight_379} \,\times\, \text{prevalence_379}}`
      -
+   * - S
+     - disability weight
+     - 0
+     -
+   * - P
+     - prevalence
+     - prevalence_c493
+     -
+   * - A
+     - prevalence
+     - 0
+     -
+   * - S
+     - prevalence
+     - 1-prevalence_493
+     - simulants not prevalent with IHD
 
```

Comment:
A typo.. 1 - prevalence_c493 under S state

Reply:
It's susceptible though. Wouldn't this make sense, then?

Reply:
@kiranklc Based on my reply above, does it make sense or do you still believe there is a typo?

Reply:
prevalence_493 is the population with IHD, so (1-prev_493) is the population without IHD.